doc/project/contributing.md

```diff
@@ -0,0 +1,60 @@
+This document provides some guidelines to making those contributions.
+Important points before getting started:
+* We consider honest feedback crucial to quality.
+  We work hard to thoroughly review changes and provide actionable feedback.
+  We do this to ensure a high quality open source design.
+* Always assume good intent.
+  Our feedback may be demanding and may even feel disheartening.
+  Again, this is to support a high quality silicon design, and we definitely appreciate all OpenTitan contributions.
+* Please be friendly and patient in your communications.
+* All OpenTitan interactions are covered by [lowRISC's code of conduct](https://www.lowrisc.org/code-of-conduct/).
+* When communicating, remember OpenTitan is a security-focused project.
+  Because of this, certain issues may need to be discussed in a small group first.
+  See the [Security Issues Process]({{<ref "#security-issues" >}}) described below for more details.
+* OpenTitan involves both hardware and software.
+  We follow a hybrid approach involving both silicon and software design practices.
+* OpenTitan is a work in progress.
+  We are always looking for ways to improve and welcome feedback on any project matter, technical or not.
+
+**Important**: Please read the next three, short sections on reporting bugs, reporting security issues, and contributing code in preparation for making your first contribution to OpenTitan.
+If you would like more details, see the Detailed Contribution Guide (coming soon).
+
+## Bug reports
+
+**To report a security issue, please follow the [Security Issues Process]({{<ref "#security-issues" >}})**.
+
+Ideally, all designs are bug free.
+Realistically, each piece of collateral in our repository is in a different state of maturity with some still under active testing and development.
+See the [Hardware Development Stages]({{< relref "development_stages" >}}) for an example of how hardware progress is tracked.
+
+We are happy to receive bug reports and eager to fix them.
+Please make reports by opening a new issue in our [GitHub issue tracker](https://github.com/lowRISC/opentitan/issues).
+
+## Security issues
+
+Security is of major importance to the OpenTitan project.
+When dealing with security matters, and in keeping with standard industry practice, there are reasons why it makes sense to be cautious and have a non-public discussion within a small group of experts before full disclosure.
+For example,
+* to ensure responsible disclosure of vulnerabilities,
+* or to discuss the security impact of new features or proposed changes to an existing feature.
+
+If you believe you have found a security issue or intend to work on potentially security sensitive matters, please first reach out to our experienced security team at security@opentitan.org before starting a public discussion.
+That will enable us to engage successfully without creating undue risk to the project or its consumers.
+
+## Contributing code
```

Comment:
This section focuses on "heavy" contributions starting with a mailing list discussion. Most contributions we see, however, are small fixes here and there. Maybe we can turn the section around, starting with the "trivial contributions" (and rename them, contributions are rarely "trivial") and tell people just to open a PR in that case?

And it would be great if we can link or mention the need for agreement to the CLA with signed-off-by lines. Neither https://docs.opentitan.org/doc/ug/github_notes/ nor https://github.com/lowRISC/opentitan/blob/master/CONTRIBUTING.md are perfect documentation on this, but at least a starting point.

Reply:
I've turned it round and added a note about the CLA (linking to CONTRIBUTING). In this PR I'd rather stick to linking to existing guidance on the CLA, as any refactoring of that text likely needs at least a quick check from the legal committee and duplicating it risks introducing unwanted divergence over time.